pkg/services/librarypanels/librarypanels_test.go

```diff
@@ -1169,8 +1115,7 @@ func TestImportLibraryPanelsForDashboard(t *testing.T) {
 				},
 				"datasource": "${DS_GDEV-TESTDATA}",
 				"libraryPanel": map[string]interface{}{
-					"uid":  outsideUID,
-					"name": outsideName,
```

Comment:
I think we want to keep `name` here for this test, since this is for the case of importing a dashboard and creating any missing library panels

Reply:
but if you read my description it automatically fills in name from UID upon importing dashboards. it adds significant complexity of hacking to workaround

Reply:
But what about the case where you're importing a dashboard (and library panel) from an external Grafana instance? The particular library panel being imported won't exist on the importing instance, so it needs to be supplied a name when the library panel is created during the import process

Reply:
I'm specifically just concerned about this one case. I agree that the dashboard JSON stored in the DB shouldn't store the library panel's name